### DISCUSSION TITLE
RC help - multiple receivers, defeat startup lockouts

### DISCUSSION THREAD

**Discussion by lewis6x** [OpenI6X/opentx#409] [Q&A] [answered]
I'm racing RC sailboats (DF65) and working on radio controlled buoys. I'm happy with the Flysky FS-i6x for sailing and had hoped to use the same TX to control and position buoys prior to racing. Unfortunately the only way to change to a different RX is to power down the currently selected RX. Obviously once a buoy is away from the dock unless I go out in a boat this is impossible. I know the safety reason for this feature but in this application it's not needed. Once the buoy is in position it will just sit there for an hour or so before needing to be piloted back to dock. 

Additionally I would like to use the three position switch for a anchor winch. The center position works great as off, up position as anchor up and down position as anchor down. Is there a way to defeat the lockout for all switches up when starting the TX?

On the sailboat I'm using FS-IA6B type receivers. I'd like to stick with those if possible. 

The questions:

- Is it possible to use OpenTX on the FS-i6x to switch models without powering down receivers?

- Is it possible to bypass the all switches up lockout?

- Would I be better off with a different TX?

- Do I just have to resign myself to carrying a sack full of TXs? One for each buoy?

TIA!

**Comment by Snekovo** [comment, marked as answer]
the receiver has the assigned number 00-15 when pairing
assign a unique receiver (iA6B) in the model settings 
set failsafe to the "receiver" option or to your own requirement
then you can switch models without having to switch off the previous receiver

switch and throttle control can be disabled in the settings "pre-flight inspection"

**Comment by lewis6x** [comment]
Thanks for the reply! Sounds promising!

Excuse my ignorance but where do I find the .bin file as described in "opentx/wiki/Flashing-&-Upgrading" "3. Choose "Local file" and drag'n'drop .bin file"?

**Reply by Snekovo** [reply]
> Thanks for the reply! Sounds promising!
> 
> Excuse my ignorance but where do I find the .bin file as described in "opentx/wiki/Flashing-&-Upgrading" "3. Choose "Local file" and drag'n'drop .bin file"?

Open transmiter and To enter DFU mode and jump two pads labeled R53 viz picture in wiki
Next Flash using EdgeTX buddy webpage

I use Flashing from Linux

Next time you can upgrade via USB storage

**Comment by lewis6x** [comment]
I understand the pin jumping and the various ways to flash the image. 

The thing I don't understand is where to get the .bin file.

**Reply by Snekovo** [reply]
get the file .bin here on git :) https://github.com/OpenI6X/opentx/releases
Download the file openi6x_1.11.1.zip and unzip it. 
It contains the file you are looking for.

**Comment by lewis6x** [comment]
I got it working!
I can switch models without powering down receivers! 
I have me curves for rudder and sheets! 
No more annoying messages on TX startup!
THANK YOU!!!

The thing that threw me for a loop was the current FS-I6x units apparently no longer have a chip that can be flashed with the STM32 stuff. I used the cable that came with the TX and a piece of software called [QMK Toolbox](https://github.com/qmk/qmk_toolbox). Worked great!

The only two items I still haven't figured out are how tsee the RX battery level on the TX screen and related how to customize the DISPLAY screens. No matter what changes I make in options 11/11 I seem to get the same 5 screens with the UP button.

1. Name of model, TX voltage, RX receive level and trims.
2. CHANNELS MONITOR
3. same as 1 but with 8 numeric values for the sticks
4. same as 1 but with 8 bar graphs showing the positions of the sticks
5. same as 1 but with switch A-D positions, bar graphs for pots and boxes with dots showing stick positions

What am I missing?

**Reply by Snekovo** [reply]
>...
> The only two items I still haven't figured out are how tsee the RX battery level on the TX screen and related how to customize the DISPLAY screens. No matter what changes I make in options 11/11 I seem to get the same 5 screens with the UP button.
> 

I'm on vacation without a transmitter.

So in short, telemetry screen -> detect sensors.
Then the A1 sensor appears which is the battery voltage.
A1 add to the telemetry display.

If you are using an external telemetry sensor the battery is then found as A3 

Petr

**Comment by lewis6x** [comment]
Thanks! Works GREAT!!! 

I was thinking of getting a fancier TX but the FS-i6X does all I need. Once I figured out the inputs->mixer->outputs concept and curves it does all I need. Being able to download via USB the EEPROM.BIN file for backups is a big plus. 

ENJOY your vacation!
